sdk/appconfiguration/azure-spring-cloud-feature-management/README.md

```diff
@@ -235,9 +235,9 @@ The following steps demonstrate an example of a progressive rollout for a new 'B
 
 #### Targeting in an Application
 
-An example web application that uses the targeting feature filter is available in the [FeatureFlagDemo](https://github.com/microsoft/spring-cloud-azure/tree/master/spring-cloud-azure-samples/feature-management-web-sample) example project.
+An example web application that uses the targeting feature filter is available in the [FeatureFlagDemo](../azure-spring-cloud-feature-management-web) example project.
```

Comment:
/CC @saragluna @conniey 
@chenrujun, why did you approve this when this link is clearly wrong? The new link for the FeatureFlagDemo doesn't even link to a sample, it's linking to the source directory for azure-spring-cloud-feature-management-web. Unless I'm mistaken this should be linking to https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-boot-samples/azure-spring-cloud-feature-management-web-sample where the sample actually is.

Reply:
@JimSuplizio .
You are right.

Originally, I thought it just change the repo from `spring-cloud-azure` to current repo, I didn't found the path error.
Thank you for pointing out.

And IMU, `FeatureFlagDemo` is a Java class name, this class does not exist any more.
So we should write like this:

```
An example web application that uses the targeting feature filter is available in the [example project](../../spring/azure-spring-boot-samples/azure-spring-cloud-feature-management-web-sample) .
```

Reply:
@JimSuplizio thanks for pointing this out. Yes, it should be pointing to the new sample project. @chenrujun I think the `FeatureFlagDemo` is the name of the sample project, but if it is confusing we could change that.

Reply:
@chenrujun @saragluna It was brought to my attention that the spring team should probably fix this. I had taken a stab at this (incorrectly). The fix can be made inline to this PR or a new one that addresses this/

Reply:
I created a new PR: https://github.com/Azure/azure-sdk-for-java/pull/14547/files